CHANGELOG.md

```diff
@@ -27,7 +27,8 @@
 
 - Update `libp2p-core`, `libp2p-floodsub`, `libp2p-gossipsub`, `libp2p-mplex`,
   `libp2p-noise`, `libp2p-plaintext`, `libp2p-pnet`, `libp2p-request-response`,
-  `libp2p-swarm`, `libp2p-tcp`, `libp2p-websocket` and `parity-multiaddr`.
+  `libp2p-swarm`, `libp2p-tcp`, `libp2p-uds`, `libp2p-websocket` and
+  `parity-multiaddr`.
```

Comment:
`0.29` is already released, so any new change should probably get a new entry.

Reply:
Good catch. Thanks @romanb!